platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/XMLUtils.java

```diff
@@ -279,41 +341,70 @@ public Transformer getXmlTransformer(boolean omitXml) throws TransformerExceptio
   }
 
   /**
-   * This class is used with the processElements method. Inside the function, set the value of the
-   * result holder. That value is then returned by the processElementsFunction.
+   * Given a TransformerFactory, sets secure settings according to OWASP guidelines
+   *
+   * @param tf TransformerFactory to be configured
    */
-  public static class ResultHolder<T> {
+  private void setSecureTransformerFactorySettings(TransformerFactory tf) {
+    tf.setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "");
```

Comment:
Why do we set these to empty strings?

Reply:
OWASP guidelines
https://www.owasp.org/index.php/XML_External_Entity_(XXE)_Prevention_Cheat_Sheet#TransformerFactory